modules/text_server_fb/gdextension_build/methods.py

```diff
@@ -73,31 +73,6 @@ def disable_warnings(self):
         self.Append(CXXFLAGS=["-w"])
 
 
-def make_icu_data(target, source, env):
```

Comment:
This one is completely redundant (fallback TS do not use ICU) and can be removed.